faqs/README.md

```diff
@@ -295,13 +301,17 @@ Do not use `rpi-update` unless you have been recommended to do so by a Raspberry
 
 Raspberry Pi do not currently provide an official 64-bit OS, for a number of reasons. Firstly, since we still sell devices that are 32-bit, we would need to support two separate distributions, and at the moment we do not have the support capacity. Secondly, building a full 64-bit OS would require a considerable amount of work to, for example, fix the interfacing to the 32-bit Videocore GPU. There are third-party 64-bit operating systems available, but they do not have the full support for the GPU that would be a requirement for an official release.
 
-### Will it run Wine or Windows, or other x86 software?
+### Can I run PC or Mac software on the Raspberry Pi?
+
+It is not possible to run Mac software on the Raspberry Pi.
+
+You cannot run Windows software *directly* on the Raspberry Pi. It is sometimes possible to use emulation software to run Windows applications on the Raspberry Pi, but the use of emulation makes it run much slower. For example, Windows 98 runs reasonably well on the Raspberry Pi using an emulator called QEMU, however more recent Windows software runs too slowly to be useful on the Raspberry Pi.
 
-In general, this is not possible with most versions of the Raspberry Pi. Some people have put Windows 3.1 on the Raspberry Pi inside an x86 CPU emulator in order to use specific applications, but trying to use a version of Windows even as recent as Windows 98 can take hours to boot into, and may take several more hours to update your cursor every time you try to move it. We don't recommend it! As of summer 2015, a version of Windows 10 is available for use on the Raspberry Pi 2 and 3. This is an entirely new version of the operating system designed exclusively for embedded use, dubbed the Windows 10 Internet of Things (IoT) Core. It does not include the user interface (shell) or the desktop operating system.
+Most Linux software will run on the Raspberry Pi, although note that since almost all Linux software is compiled for PCs, you need to find either an ARM-specific version of the software, or have access to the source code so that you can compile it yourself.
```

Comment:
Sort of implies that ALL software needs to be found or compiled - but in fact almost everything is already in the repositories, which is easier to get at that this paragraph implies.